nerfstudio/scripts/render.py

```diff
@@ -442,6 +470,8 @@ class RenderCameraPath(BaseRender):
     """Filename of the camera path to render."""
     output_format: Literal["images", "video"] = "video"
     """How to save output data."""
+    complete: bool = True
```

Comment:
Same with this, it seems like complete should be a `@property` that gets managed internally

Reply:
I'm not really following the connection to `@property`

Reply:
I can hack around this too by suppressing? I think as a field it does need to still exist for the progress(notification) update.